# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../git/git-no-install-Error.pm.patch
# Copyright (C) 2011 - 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not install the private Error.pm

--- a/perl/Makefile.PL	2014-02-02 19:56:05.318198022 +0100
+++ b/perl/Makefile.PL	2014-02-02 19:59:38.331137009 +0100
@@ -45,7 +45,7 @@
 # We come with our own bundled Error.pm. It's not in the set of default
 # Perl modules so install it if it's not available on the system yet.
 if ( !eval { require Error } || $Error::VERSION < 0.15009) {
-	$pm{'private-Error.pm'} = '$(INST_LIBDIR)/Error.pm';
+#	$pm{'private-Error.pm'} = '$(INST_LIBDIR)/Error.pm';
 }
 
 # redirect stdout, otherwise the message "Writing perl.mak for Git"
